src/connectors/numeric-menu/connectNumericMenu.ts

```diff
@@ -233,12 +248,20 @@ export default function connectNumericMenu(renderFn, unmountFn = noop) {
 
         return withMaxRefinement;
       },
+
+      // @TODO: remove this (used in widget test now)
+      _getRefine() {
+        return connectorState.refine;
+      },
     };
   };
-}
+};
 
-function isRefined(state, attribute, option) {
-  const currentRefinements = state.getNumericRefinements(attribute);
+function isRefined(state: SearchParameters, attribute: string, option: Option) {
+  // @TODO: same as another spot, why is this mixing arrays & elements?
+  const currentRefinements = state.getNumericRefinements(
```

Comment:
getNumericRefinements returns an array (according to typings), but we use it as a single OperatorList, which one is correct?